en/technical-reference/going-live.md

```diff
@@ -3,30 +3,37 @@
 
 This document is designed to assist you in switching from *Sandbox* environment to *Production* environment, in order to start accepting real world payment for your business. This is also known as "Going Live".
 
+?>Make sure you have already completed the test payment using our [Payment Simulator](/en/technical-reference/sandbox-test.md) in the Sandbox environment to ensure that your implementation is working fine.
 
-## 1. Login to Your Midtrans Production Environment Dashboard
+## 1. Checking Your Production Account
+To make sure your payment channels on Production account is already active, follow the steps given below.
```

Comment:
> payment channels

Is an internal wording. to talk with external audience we use `Payment Methods`. The term has been used all over the docs too, so you should just need to be consistent with the term.

Reply:
better wording:
> To make sure that your Midtrans account already has activated payment method(s) on Production Environment mode, follow the steps given below.